src/olympia/accounts/views.py

```diff
@@ -774,3 +778,88 @@ def post(self, request):
                 _('Notification [%s] does not exist') % notification_name
             )
         return Response(serializer.data)
+
+
+class FxaNotificationView(FxAConfigMixin, APIView):
+    authentication_classes = []  # TODO: reimplement FxA auth in authentication_class?
```

Comment:
The one thing it could bring us would be sharing some code with the auth class that already deals with JWT... though I don't think there would be much besides parsing of the header, since we have special considerations for jwts crafted by developers with API keys which is completely different from dealing with FxA